### DISCUSSION TITLE
Internal server error while creating new mail server

### DISCUSSION THREAD

**Discussion by tricelex** [postalserver/postal#1618] [Help with using Postal] [answered]
I keep getting Internal server error when creating a new mail server, 
Nothing really helpful from the postal web container log.

**Comment by willpower232** [comment]
I'm afraid the only information is in that log. What do you see?

**Comment by tricelex** [comment, marked as answer]
Yeah, I've fixed it, had to check the logs inside the docker container.
It was simply wrong database information😅
